tests/030_views/031_vcard.py

```diff
@@ -0,0 +1,12 @@
+from qr_vcard.factories import VCardFactory
+
+
+def test_blog_index_empty(db, client):
+    """
+    Without any existing blog, index view should just return the empty text.
+    """
+    vcard = VCardFactory()
+
+    response = client.get(vcard.get_vcf_url())
```

Comment:
```suggestion
    response = client.get(VCard.get_vcf_url())
```
More imports, so less performance.
But, it get clearer that bet_vcf_url() is a VCard method.

Reply:
VCard is already your model name and even it is not imported here you could need it in future, or using this part of code in another one and forget it which could result to uncanny bugs due to the fact you override VCard name.

Also in Python we commonly avoid uppercase in non top level variable names, try to stand to some conventions for more readability, you will prevent you from many headaches;